app/server/src/features/accounts/methods.ts

```diff
@@ -209,4 +211,57 @@ export async function deleteAccount(prisma: PrismaClient, input: DeleteAccountFo
 
     // return deleted user
     return { deletedUser };
-}
\ No newline at end of file
+}
+
+export async function resetPasswordRequest(prisma: PrismaClient, input: ResetPasswordRequestForm) {
+    const result = await prisma.user.findUnique({
+        where: { email: input.email }
+    });
+
+    const accountFound = !!result;
+
+    // No need to send email if the account does not exist
+    if (!accountFound) throw new Error('No account with that email was found');
+
+    const token = await jwt.sign({ email: input.email });
+    const passwordResetLink = `prytaneum.io/reset-password?token=${token}`
```

Comment:
is the https:// missing?  Also, do we want to hard code the prytaneum.io?  Maybe make it env variable (PUBLIC_URL)?